src/api/cvc4cppkind.h

```diff
@@ -1852,6 +1852,18 @@ enum CVC4_PUBLIC Kind : int32_t
    *   mkTerm(Kind kind, const std::vector<Term>& children)
    */
   COMPREHENSION,
+  /**
+   * Returns an element from a given set.
+   * If a set A = {x}, then the term (choose A) is equivalent to the term x.
+   * If the set is empty, then the atom that uses (choose A) is false.
```

Comment:
If the set is empty, then (choose A) is an arbitrary element.

Reply:
done